app/src/Command/TruncateMessageCommand.php

```diff
@@ -12,7 +13,8 @@
 
 #[AsCommand(
     name: 'agentj:truncate-message-since-days',
-    description: 'Truncate tables msgs and msgrcpt since X days in argument, X more than 10, by default it is 30 days',
+    description: 'Truncate tables msgs, quarantine, out_msgs and out_quarantine
+     since X days in argument, X more than 10, by default it is 30 days',
```

Comment:
Ça fait une description un peu longue et trop détaillée. Je propose :


```suggestion
    description: 'Truncate incoming and outgoing messages older than a number of days (minimum 10, 30 by default)',
```